docs/src/cli/app-updates.md

```
@@ -0,0 +1,137 @@
+      "source": "npm",
+      "package": "myapp",
+      "command": "self-update"
+    }
+  }
+}
+```
+
+That is enough. Your app now checks npm for a newer `myapp`, and when there is
+one it prints two lines to stderr:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)